admin-app/src/components/rounds/AddEditRoundFormModal.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div>
+    <a-form :form="form" @submit="handleSubmitAction">
+      <a-form-item v-if="errorMessage.length">
+        <a-alert type="error" :message="errorMessage" />
+      </a-form-item>
+      <a-form-item>
+        <a-input v-decorator="[
+            'id'
+          ]" hidden />
+      </a-form-item>
+      <a-form-item label="Name" :label-col="{ span: 5 }" :wrapper-col="{ span: 12 }">
+        <a-input
+          v-decorator="[
+            'name',
+            {rules: [{ required: true, message: 'Please input the name!' }]}
+          ]"
+        />
+      </a-form-item>
+      <a-form-item label="Short name" :label-col="{ span: 5 }" :wrapper-col="{ span: 12 }">
+        <a-input
+          v-decorator="[
+            'shortName',
+            {rules: [{ required: true, message: 'Please input the short name!' }]}
+          ]"
+        />
+      </a-form-item>
+      <a-form-item label="Sort order" :label-col="{ span: 5 }" :wrapper-col="{ span: 12 }">
+        <a-input-number :min="1"
+          v-decorator="[
+            'sortOrder',
+            {rules: [{ required: true, message: 'Please input the sort order!' }]}
+          ]"
+        />
+      </a-form-item>
+      <a-form-item :wrapper-col="{ span: 12, offset: 5 }">
+        <a-button type="primary" html-type="submit">Save</a-button>
+        <a-button :style="{ marginLeft: '8px' }" @click="handleCancel">Cancel</a-button>
+      </a-form-item>
+    </a-form>
+  </div>
+</template>
+
+<script lang="ts">
+import { mapState } from 'vuex';
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+import RoundInterface from '@/domains/models/Round';
+
+@Component({
+  computed: mapState('rounds', ['editData'])
+})
+export default class AddEditRoundFormModal extends Vue {
+  public editData!: RoundInterface;
+  private form: any;
+  private errorMessage: string = '';
+
+  private created() {
+    this.form = this.$form.createForm(this, {
+      mapPropsToFields: () => {
+        let id = 0;
+        let name = '';
+        let shortName = '';
+        let sortOrder = 0;
+
+        if (this.editData) {
+          id = this.editData.id;
+          name = this.editData.name;
+          shortName = this.editData.shortName;
+          sortOrder = this.editData.sortOrder;
+        }
+
+        return {
+          id: this.$form.createFormField({
+            value: id
```

Comment:
`const {id,name,shortName,sortOrder}=this.editData;`
````suggestion
            value: id||0
```